### DISCUSSION TITLE
RX-V473 works!

### DISCUSSION THREAD

**Discussion by htobbe** [mvdwetering/yamaha_ynca#234] [General]
I'm able to power on/off, (un)muting, volume control and input selection (not AUDIO)

**Comment by mvdwetering** [comment]
Thanks for the confirmation.

Interestingly I just built a workaround for the AUDIO input for RX-V475 (still unreleased).

I might need to make the workaround user configurable to easily apply it to multiple receiver models if it is the same issue.
Could you download the diagnostics and attach the file here so I can see if it is the same?

![image](https://github.com/mvdwetering/yamaha_ynca/assets/732514/c8a1fa98-08cc-4b32-b6a0-2104e564169d)

**Comment by mvdwetering** [comment]
The AUDIO input should work in v7.6.0
